blocks/core/ff_module/ff_module-form-marking-controls/ff_module-form-marking-controls.js

```diff
@@ -19,7 +19,7 @@ module.exports = React.createClass({
 	},
 	render: function() {
 		return <div className={this.generateClass('ff_module-form-marking-controls', this.props.modifier)}>
-			<ControlBar>
+			<ControlBar modifier={this.props.controlBar.modifier}>
```

Comment:
Again, I'd change this to `<ControlBar {...this.props.controlBar}>` so you can pass all the props that ControlBar uses without needing to redefine every time :)